src/view/new-event.js

```diff
@@ -0,0 +1,375 @@
+import { humanizeEventDate } from '../utils/date.js';
+import { getLastWord, getOffersByType, getDestinationByName} from '../utils/common.js';
+import Smart from './smart.js';
+
+import flatpickr from 'flatpickr';
+
+import '../../node_modules/flatpickr/dist/flatpickr.min.css';
```

Comment:
А так точно можно?)
```suggestion
import 'flatpickr/dist/flatpickr.min.css';
```
попробуй